lightning/src/ln/reorg_tests.rs

```diff
@@ -212,11 +212,7 @@ fn do_test_unconf_chan(reload_node: bool, reorg_after_reload: bool, use_funding_
 		} else {
 			disconnect_all_blocks(&nodes[0]);
 		}
-		if connect_style == ConnectStyle::FullBlockViaListen && !use_funding_unconfirmed {
```

Comment:
I don't think the `connect_style` parameter is used anymore in this test, everything passes when it's `unused`

Reply:
All tests pass with that line commented out